src/update-files.ts

```diff
@@ -44,6 +44,7 @@ async function updateFile(relPath: string, operation: Operation): Promise<boolea
     case 'bower.json':
     case 'component.json':
     case 'jsr.json':
+    case 'deno.json':
```

Comment:
Is here missing `deno.jsonc`?

Reply:
`jsr.jsonc` isn't here.
I think `jsonc` and `json` are differnt

Reply:
I am not sure if it was correct. But let's have it for now, as jsonc would be a bit tricky to deal with correctly and I'd have the bandwidth to look into it right now.

Reply:
When I found a bug for that, I'll contribute it again!